tests/regression/37-congruence/03-interval-overflow.c

```diff
@@ -0,0 +1,44 @@
+// PARAM: --enable ana.int.interval --enable ana.int.congruence --disable ana.int.def_exc
+// Overflow information should be passed from the interval domain to congruences
+#include <assert.h>
+#include <stdio.h>
+
+int main(){
+  char r;
+
+  if (r) {
+    r = -68;
+  } else {
+    r = -63;
+  }
+
+  char k = r - 80;
+  assert (k == 0); //UNKNOWN!
+
+  char non_ov = r - 10;
+  assert (non_ov == -78); //UNKNOWN
```

Comment:
Is `UNKNOWN` the right annotation here?

Reply:
No, it should have been UNKNOWN! due to the branching above. I fixed it